cmake/modules/AddSwift.cmake

```diff
@@ -477,7 +477,9 @@ function(add_swift_host_library name)
   endif()
 
   add_library(${name} ${libkind} ${ASHL_SOURCES})
-  add_dependencies(${name} ${LLVM_COMMON_DEPENDS})
+  if (LLVM_COMMON_DEPENDS)
```

Comment:
Can you add a comment here describing what LLVM_COMMON_DEPENDS is and maybe provide a reference to the large comment in ./lib/CMakeLists.txt where we describe it? I just don't want this to be mysterious.

Reply:
you already did this. Thanks